runtime/src/main/java/com/redhat/ipaas/runtime/db/entity/ConfiguredConnector.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.runtime.db.entity;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity(name="Connector")
+@Table(name="ipaas_connector")
+public class ConfiguredConnector {
+
+    @Id
+	@Getter @Setter @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    @Getter @Setter
+    private String name;
+    @Getter @Setter
+    private String tags;
+    @Getter @Setter @ManyToOne @JoinColumn(name = "connector_id")
+    private Connector connector;
+    @Getter @Setter @OneToMany(fetch = FetchType.LAZY)
+    private List<ActionInstance> actionInstances;
```

Comment:
Yeah not sure, but I thought we need to access the connector.connectorProperties when working with the ConfiguredConnector.